modules/launchd/launchd.nix

```
@@ -121,7 +121,18 @@ in
         key. Please note: many people are confused by this key. Please read `execvp(3)` very carefully!
       '';
       # TODO: Remove this some time after 25.11.
-      apply = map (value: lib.warnIf (lib.hasInfix "&amp;" value) "A value for `ProgramArguments` contains the literal string `&amp;`. This is no longer necessary and will lead to double-escaping, as nix-darwin now automatically escapes special characters." value);
+      apply =
+        val:
+        if (builtins.isNull val) then
+          val
+        else
+          (map (
+            item:
+            lib.warnIf (lib.hasInfix "&amp;" item) ''
+              A value for `ProgramArguments` contains the literal string `&amp;`. This is no longer necessary and
+              will lead to double-escaping, as nix-darwin now automatically escapes special characters.
```

Can you change the warning string back to a one-line string?
```suggestion
              A value for `ProgramArguments` contains the literal string `&amp;`. This is no longer necessary and will lead to double-escaping, as nix-darwin now automatically escapes special characters.
```

---

Happy to, I am trying to get used to this as a best practice. However, in this case I was trying to match the surrounding formatting -- all analogous strings are hard-wrapped at roughly 120 chars.

Might it be worth standardizing on the "one sentence per line" format, which has been recommended to me by the nix-documentation (forget the official name) team?

---

Actually, the others are descriptions. I suspect this has to do with hard wraps in the warning resulting in weird linebreaks at runtime.

---

We do have some other warnings that are manually split onto multiple lines, so I agree that we should standardize at some point. For now I'm just trying to have as small an impact as possible